gtk4/Gir.toml

```diff
@@ -186,6 +185,7 @@ generate = [
 
 # TODO
 ignore = [
+    "Gtk.CustomLayout",
```

Comment:
This needs a comment to that tells why it's ignored, so something like ` # subclass LayoutManager if you need a custom layout`

Reply:
also a rebase

Reply:
Done.